packages/map-template/src/components/Search/Categories/Categories.jsx

```diff
@@ -136,14 +141,14 @@ function Categories({ onSetSize, getFilteredLocations, searchFieldRef }) {
     }, [selectedCategory]);
 
     /*
-     * Use the active category state to scroll into view
-     * and center the category.
+     * If the active category is a prop/query parameter and the bottom sheet is loaded, 
+     * then scroll into view and center the active category. 
      */
     useEffect(() => {
-        if (activeCategory) {
+        if (activeCategory && hasCategoryProp && isBottomSheetLoaded) {
```

Comment:
Will this work on larger screens where the bottom sheet is not used?

Reply:
Good catch. I adjusted the check to also consider the desktop screens where there's no bottom sheet.